dask/dataframe/io/parquet/arrow.py

```diff
@@ -138,15 +184,30 @@ def read_metadata(
         parts, dataset = _determine_dataset_parts(
             fs, paths, gather_statistics, filters, kwargs.get("dataset", {})
         )
+        col_chunk_paths = False
```

Comment:
Might be worth to add a short comment here why this is done. My understanding is that this checks if the file paths are correctly set in the `_metadata` file (so which was not the case with dask+pyarrow right now)

Reply:
Ah, I see there is some comment below